ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -472,8 +476,22 @@ def run_ocs_upgrade(operation=None, *operation_args, **operation_kwargs):
     )
 
     # For external cluster , create the secrets if upgraded version is >= 4.8
-    if config.DEPLOYMENT["external_mode"] and upgrade_version >= "4.8":
-        create_external_secret(ocs_version=upgrade_version, apply=True)
+    if (
+        config.DEPLOYMENT["external_mode"]
+        and original_ocs_version == "4.7"
+        and upgrade_version == "4.8"
```

Comment:
The conditional statement is not matching with the above comment which says upgraded version is >= 4.8

Reply:
we need update new credentions only when we upgrade from OCS 4.7 to OCS 4.8

Reply:
changed the comment